src/dxtbx/format/FormatROD.py

```diff
@@ -482,6 +484,157 @@ def decode_TY6_oneline(self, linedata, w):
         return ret
 
 
+class FormatROD_Arc(FormatROD):
+    @staticmethod
+    def understand(image_file):
+        offset = 256
+        general_nbytes = 512
+        with FormatROD_Arc.open_file(image_file, "rb") as f:
+            f.seek(offset + general_nbytes + 548)
+            detector_type = struct.unpack("<l", f.read(4))[0]
+            if detector_type in [12, 14]:
+                return True
+        return False
+
+    def _detector(self):
+        """2 or 3 panel detector, each rotated 38° from its neighbour."""
+
+        theta_rad = self._gonio_start_angles[1] / 180 * np.pi
+        detector_rotns_rad = np.array(self._bin_header["detector_rotns"]) / 180 * np.pi
+        rot_e1 = np.array(
+            r3_rotation_axis_and_angle_as_matrix([0, 0, 1], detector_rotns_rad[0])
+        ).reshape(3, 3)  # clockwise along e1 = Z
+        rot_e2 = np.array(
+            r3_rotation_axis_and_angle_as_matrix([-1, 0, 0], detector_rotns_rad[1])
+        ).reshape(3, 3)  # ANTI-clockwise along e2 = X
+        rot_theta = np.array(
+            r3_rotation_axis_and_angle_as_matrix([0, -1, 0], theta_rad)
+        ).reshape(3, 3)  # ANTI-clockwise along e3 = Y
+        detector_axes = rot_theta.dot(rot_e2.dot(rot_e1))
+
+        pixel_size_x = self._bin_header["real_px_size_x"]
+        pixel_size_y = self._bin_header["real_px_size_y"]
+        origin_at_zero = np.array(
+            [
+                -self._bin_header["origin_px_x"] * pixel_size_x,
+                -self._bin_header["origin_px_y"] * pixel_size_y,
+                -self._bin_header["distance_mm"],
+            ]
+        )
+        # Get origin for the first panel, not rotated
+        origin = detector_axes.dot(origin_at_zero)
+
+        d = Detector()
+
+        root = d.hierarchy()
+        root.set_local_frame(detector_axes[:, 0], detector_axes[:, 1], origin)
+
+        self.coords = {}
+        panel_idx = 0
+
+        pnl_data = []
+        gap_px = 30
+        pnl_data.append({"xmin": 0, "ymin": 0, "xmax": 385, "ymax": 775, "xmin_mm": 0})
+        pnl_data.append(
+            {
+                "xmin": (385 + gap_px),
+                "ymin": 0,
+                "xmax": 800,
+                "ymax": 775,
+                "xmin_mm": (385 + gap_px) * pixel_size_x,
+            }
```

Comment:
Aren't these constants available in the image header?

Reply:
Yes, the Windows CAP program gives me useful values, which it suggests are in the header:
```
psImage->sextra.smultimodule_imgdef
  ixmodules_imgheader: 3
  iymodules_imgheader: 1
  imodule_nx_imgheader: 385
  imodule_ny_imgheader: 775
  imodule_gapx_imgheader: 30
  imodule_gapy_imgheader: 0
```
I'll try to find them.

Reply:
These come in a struct called `campar_extra_tag`, but there may be some seeking between these, so I'm not sure if it is contiguous with earlier parts of the header. To find out, I'll have to dig back into the source code